docs/getting-started/advanced-setting.asciidoc

```diff
@@ -71,9 +71,9 @@ collectors to ship data to {es}, the data must be mapped to ECS.
 [[update-threat-intel-indices]]
 == Update default Elastic Security threat intelligence indices
 
-The `securitySolution:defaultThreatIndex` advanced setting defines threat intelligence indices that {elastic-sec} will use when collecting threat indicators. The setting controls features that query threat indices, such as the Threat Intelligence view on the Overview page and threat indicator match rules. One or more threat intelligence indices can be defined; the `filebeat-*` index is specified by default.
+The `securitySolution:defaultThreatIndex` advanced setting defines threat intelligence indices that {elastic-sec} features use when querying for ingested threat indicators. The setting controls features that query threat intelligence indices, such as the Threat Intelligence view on the Overview page and indicator match rules. One or more threat intelligence indices can be defined; the `logs-ti*` index pattern is defined by default and used by {agent} integrations.
 
-IMPORTANT: Threat intelligence indices aren't required to be ECS compatible when used in an indicator match rule. However, we recommend compatibility if you'd like for your alerts to be enriched with relevant threat indicator information. By default, indicator match rules are configured to ingest indicators from the Threat Intel {filebeat} module and to check for fields prefixed with `threatintel.indicator`. See {filebeat-ref}/filebeat-module-threatintel.html[Threat Intel module] for more information about these fields. This rule setting is stored in the *Indicator prefix override* field, under indicator match rule advanced settings. If your indicator data contains indicator fields at a different location, update this field accordingly to ensure alert enrichment can still be performed.
+IMPORTANT: Threat intelligence indices aren't required to be ECS compatible when used in an indicator match rule. However, we recommend compatibility if you'd like for your alerts to be enriched with relevant threat indicator information. When querying for threat indicator data, indicator match rules uses the threat indicator path specified in the *Indicator prefix override* advanced setting. By default, this setting looks for fields prefixed by `threat.indicator`. More information about this setting is available at <<rule-ui-advanced-params, Configure advanced rule settings>>.
```

Comment:
Second sentence: I'd dare to say that we **strongly** recommend ECS compatibility (we have some experimental features in the pipeline that may work with ECS threat indices only)